app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -6,7 +6,16 @@ type Query {
 }
 
 type Customer {
-    wishlist: Wishlist! @resolver(class:"\\Magento\\WishlistGraphQl\\Model\\Resolver\\CustomerWishlistResolver") @doc(description: "Contains the contents of a customer's wish lists")  @cache(cacheable: false)
+    wishlists(
+        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. This attribute is optional."),
+        currentPage: Int = 1 @doc(description: "Specifies which page of results to return. The default value is 1.")
+    ): WishlistsOutput @doc(description: "All wishlists for customer.") @resolver(class:"\\Magento\\WishlistGraphQl\\Model\\Resolver\\CustomerWishlists")
```

Comment:
@eduard13 👍 for adding pagination. Can you update the queryname and description based on the latest architecture. https://github.com/magento/architecture/blob/43f4fac77df50ae59954af27c8e5fb2bf034282f/design-documents/graph-ql/coverage/Wishlist.graphqls#L12